---
title: LOCALDB_ERROR_INTERNAL_ERROR | Microsoft Docs
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
author: stevestein
ms.author: sstein
manager: craigg
ms.openlocfilehash: 93af9ff4c28570f1a7304ff4255c876c79ba5ee1
ms.sourcegitcommit: 61381ef939415fe019285def9450d7583df1fed0
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 10/01/2018
ms.locfileid: "47844327"
---
# <a name="localdberrorinternalerror"></a>LOCALDB_ERROR_INTERNAL_ERROR
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Detalles  
  
|||  
|-|-|  
|Nombre del producto|SQL Server|  
|Identificador del evento|264|  
|Origen del evento|SQL Server Local Database Runtime 12.0|  
|Componente|API de Local Database Runtime|  
|Texto del mensaje|Error inesperado en una llamada al método de API de la instancia de Local Database. Vea el registro de aplicación de Windows para obtener los detalles del error.|  
  
## <a name="explanation"></a>Explicación  
 Error inesperado.  
  
## <a name="user-action"></a>Acción del usuario  
 Vea el registro de eventos para obtener detalles.  
  
  
